CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [1.26.3] - 2021-03-04
```

Comment:
Should this be a minor version change?

Reply:
You'll also need to update check-compatibility.js and package.json version number.

Reply:
Done.